sensei/CatalystAnalysisAdaptor.h

```diff
@@ -48,7 +48,7 @@ class SENSEI_EXPORT CatalystAnalysisAdaptor : public AnalysisAdaptor
    */
   virtual void AddPythonScriptPipeline(const std::string& fileName,
     const std::string& resultProducer, const std::string& steerableSourceType,
-    const std::string& resultMesh, int scriptVersion = 1);
+    const std::string& resultMesh, int versionHint = 2);
```

Comment:
Should this 1, as SENSEI does not support Catalyst 2 yet. This breaks oscillator/catalyst-steering. Setting it back to 1 fixes it.